weave/trace_server_bindings/stainless_remote_http_trace_server.py

```diff
@@ -0,0 +1,2065 @@
+import datetime
```

Comment:
it seems like a huge amount of this is copy/pasta. did you consider creating a more reusable adapter pattern?